inyoka_theme_default/static/style/inyoka/inyoka.less

```diff
@@ -125,47 +143,51 @@ html, body {
 }
 
 .sidebar {
-  &:extend(.panel);
-  &:extend(.panel-default);
+  .make-sm-column(3);
+
+  &-group {
+    &:extend(.panel);
+    &:extend(.panel-default);
+
+    &-danger {
+      &:extend(.panel-danger);
 
-  &-danger {
-    &:extend(.panel-danger);
+      .sidebar-group-heading {
+        &:extend(.panel-heading);
+        &:extend(.panel-danger > .panel-heading);
+
+        & > h1 {
+          &:extend(.panel-title);
+        }
+      }
+    }
 
-    .sidebar-heading {
+    &-heading {
       &:extend(.panel-heading);
-      &:extend(.panel-danger > .panel-heading);
+      &:extend(.panel-default > .panel-heading);
 
       & > h1 {
         &:extend(.panel-title);
       }
     }
-  }
-
-  &-heading {
-    &:extend(.panel-heading);
-    &:extend(.panel-default > .panel-heading);
 
-    & > h1 {
-      &:extend(.panel-title);
+    & > ul {
+      &:extend(.list-group);
+      &:extend(.panel > .list-group);
     }
-  }
-
-  & > ul {
-    &:extend(.list-group);
-    &:extend(.panel > .list-group);
-  }
 
-  &-item {
-    &:extend(.list-group-item);
-    &:extend(.panel > .list-group .list-group-item);
+    &-item {
+      &:extend(.list-group-item);
```

Comment:
imho its hard to use extends all, as its not only .list-group-item and also .panel involved